Mlem/Views/Tabs/Settings/Contributors/Contributors View.swift

```diff
@@ -17,7 +17,7 @@ struct ContributorsView: View
     {
         VStack(alignment: .center, spacing: 10)
         {
-            CachedAsyncImage(url: contributor.avatarLink)
+            CachedAsyncImage(url: contributor.avatarLink, urlCache: AppConstants.urlCache)
```

Comment:
Nothing to change for this PR here, but we may want to build a small wrapper around `CachedAsyncImage` which automatically sets our cache to save having to remember to do it everywhere going forward?